classes/core/PKPRequest.inc.php

```diff
@@ -542,6 +542,14 @@ public function isBot()
         return $this->_isBot;
     }
 
+    /**
+     * Check if the HTTP_DNT (Do Not Track) is set
+     */
+    public function isDNTSet(): bool
+    {
+        return (array_key_exists('HTTP_DNT', $_SERVER) && ((int) $_SERVER['HTTP_DNT'] === 1));
```

Comment:
I would guess that `HTTP_DNT` is probbaly in `$_SERVER` as a `string`. I would suggest using...
```php
return (bool) $_SERVER['HTTP_DNT'] ?? false;
```